<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WKSPUN05GTR0030" failures="1" tests="2" name="Default test" time="201.813" errors="0" timestamp="2025-05-27T15:41:22 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="178.587"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="22.84">
    <failure type="java.lang.AssertionError" message="Expected login to succeed, but it failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected login to succeed, but it failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at booksWagon.StepDefinition.LoginStepDefinition.fetchDataAndLogin(LoginStepDefinition.java:194)
at âœ½.User fetches login data from Excel and performs login(file:///C:/Users/rudra.singh/Downloads/MainProj-main%20(1)/MainProj-main/BooksWagon/BooksWagon/src/test/resources/Features/02_Login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
